src/components/FarmIssueAnnouncement.tsx

```diff
@@ -161,9 +175,15 @@ const FarmIssueAnnouncement: React.FC<Props> = ({ isEnded }) => {
           </Text>
         </Flex>
 
-        <ButtonText onClick={() => setShow(prev => !prev)} style={{ color: theme.subText }}>
-          <DropdownSVG style={{ transform: `rotate(${!show ? 0 : '180deg'})` }} />
-        </ButtonText>
+        <Flex
+          sx={{
+            flex: '0 0 24px',
+            justifyContent: 'center',
+            alignItems: 'center',
+          }}
+        >
+          <DropdownIcon data-flip={show} />
```

Comment:
there is no problem, but  I prefer using 
(1) `<DropdownIcon flip={show} />` 
than using 
(2) `<DropdownIcon data-flip={show} />`
Because option 2 will have type checking for us.

```
const DropdownIcon = styled(DropdownSVG)<{flip: boolean}>`
  transition: transform 300ms;
  color: ${({ theme }) => theme.subText};
  color: ${({ theme,flip }) => flip && 'transform: rotate(180deg'};
`
```

Reply:
Sounds good. Both have pros and cons